packages/graphql-codegen-cli/src/config.ts

```diff
@@ -180,7 +180,7 @@ export function parseArgv(argv = process.argv): YamlCliFlags {
 
 export async function createContext(cliFlags: YamlCliFlags = parseArgv(process.argv)): Promise<CodegenContext> {
   if (cliFlags.require && cliFlags.require.length > 0) {
-    await Promise.all(cliFlags.require.map(mod => import(require.resolve(mod, { paths: [process.cwd()] }))));
+    await Promise.all(cliFlags.require.map(mod => import(mod)));
```

Comment:
This doesn't behave the same, it will resolve `mod` from the CLI instead of the `cwd`, you can use `createRequire` instead of `require.resolve`.

Ref https://github.com/dotansimha/graphql-code-generator/pull/5032 and https://github.com/dotansimha/graphql-code-generator/pull/4983

Reply:
@merceyz thanks 👍 https://github.com/dotansimha/graphql-code-generator/pull/6102/commits/9b55a9210b170d2438d034b6cd45f93d51a5fefa